pkg/apis/storage/types.go

```diff
@@ -335,6 +335,9 @@ type CSINodeDriver struct {
 	// understand, e.g. "nodeA" instead of "node1". This field is required.
 	NodeID string
 
+	// allocatable represents the resources of a node that are available for scheduling.
+	Allocatable VolumeLimits
```

Comment:
Since `Allocatable` is a optional field (for now), I think this should be made a pointer?

Reply:
Seems  pointer is better